### ISSUE TITLE
design / selected color isn't clear

### ISSUE THREAD

**Issue by ofZach** [ofZach/pgElectron#14]
here, in one case grey = selected, while in another case, white = selected.  needs maybe some consistency?

<img width="467" alt="screen shot 2015-08-22 at 10 01 11 am" src="https://cloud.githubusercontent.com/assets/142897/9424286/d5e8112e-48b4-11e5-957f-e5a1df6f14f5.png">

**Comment by Daandelange** [comment]
Yes, this is not clear and was the first thing I saw after this update.
Dark grey with white text for selected seems better

**Comment by Daandelange** [comment]
Btw I think I fixed this in 002c0759bdcdbc690f3a8c3bc584545bd4ffd443 (makes it white on gray)
Approve ?

**Comment by ofZach** [comment]
I still think it's confusing, I'm working on styling the select2 it might help.   The problem is that the select 2 grey is the same as the unselected platform, so for me the feels it's a bit off, like I can't tell what is active.  I'll work on this a bit, thanks for the changes!

<img width="273" alt="screen shot 2015-08-22 at 3 41 48 pm" src="https://cloud.githubusercontent.com/assets/142897/9425621/6a5dbc08-48e4-11e5-8941-e255c7f7542d.png">

**Comment by kylemcdonald** [comment]
if it helps, it was also unclear to me which one was "selected". a color might not be enough, but adding a border to the selected ones would feel very clear to me.

**Comment by ofZach** [comment]
I also think an "x" image, like the PG roadmap mockup and the adddons multiselect would be useful, ie, signaling an unselect...   I tried to do this via css but I think it has to be done with javascript.

**Comment by Daandelange** [comment]
I did a new try, might be better but I'm still not convinced.

**Comment by ofZach** [comment]
I think if the x was bigger (like the PG mockup) and it was visibility:none on unselected (so there wasn't a gap) it might help.  Obviously things will reflow as you select them, but it feels like it would be more obvious.

**Comment by Daandelange** [comment]
I made the gap on purpose so the buttons don't move on enabling/disabling them. _(because adding the "x" adds more width to the buttons)_

**Comment by ofZach** [comment]
I think it's fine if they reflow as you select them...

**Comment by Daandelange** [comment]
Should be better now :)
Also removed the "x" and the pointer when it's the only selected platform.

**Comment by kylemcdonald** [comment]
this could be closed i think, as the design is much clearer now.
